tokio/src/park/thread.rs

```diff
@@ -10,13 +10,7 @@ pub(crate) struct ParkThread {
     inner: Arc<Inner>,
 }
 
-/// Error returned by `ParkThread`
-///
-/// This currently is never returned, but might at some point in the future.
-#[derive(Debug)]
-pub(crate) struct ParkError {
```

Comment:
Reshuffling internal code. This *used* to be public, which is why `ParkError` is opaque.